mala/network/multi_predictor.py

```diff
@@ -0,0 +1,342 @@
+"""Tester class for testing a network."""
+import glob
+import os
+from zipfile import ZipFile, ZIP_STORED
+
+import ase.io
+try:
+    import horovod.torch as hvd
+except ModuleNotFoundError:
+    # Warning is thrown by Parameters class
+    pass
+import numpy as np
+import torch
+
+from mala.common.parallelizer import printout, get_rank, barrier
+from mala.network.predictor import Predictor
+from mala.network.network import Network
+from mala.datahandling.data_scaler import DataScaler
+from mala.datahandling.data_handler import DataHandler
+from mala import Parameters
+
+
+class MultiPredictor(Predictor):
+    """
+    A class for running predictions using a neural network.
+
+    It enables production-level inference.
+
+    Parameters
+    ----------
+    params : mala.common.parametes.Parameters
+        Parameters used to create this Predictor object.
+
+    networks : list
+        List of networks used for predictions.
+
+    data : mala.datahandling.data_handler.DataHandler
+        DataHandler, in this case not directly holding data, but serving
+        as an interface to Target and Descriptor objects.
+    """
+
+    def __init__(self, params, networks, data):
+        # copy the parameters into the class.
+        super(MultiPredictor, self).__init__(params, networks[0], data)
```

Comment:
`super().__init__(...)` does it.

Reply:
See above - I thought this was the correct way of doing things.

Reply:
Same as above, not wrong but a matter of preference.